src/NuGet.Clients/NuGet.Tools/NuGetUIFactory.cs

```diff
@@ -60,10 +63,18 @@ internal sealed class NuGetUIFactory : INuGetUIFactory
             INuGetUILogger logger,
             ISourceControlManagerProvider sourceControlManagerProvider)
         {
+            var signedPackageVerifier = new PackageSignatureVerifier(SignatureVerificationProviderFactory.GetSignatureVerificationProviders());
```

Comment:
Why didn't any tests catch this?

Reply:
I'm surprised why CTI team didn't catch it? it's high time we get details about what these guys tests and how...

@rrelyea @DoRonMotter

Reply:
Did you mean, "provide them details on what they should be testing and how"? 😄

Reply:
:) I mean first get the details and then see, what needs to be corrected in there. For now, we've (or at least me) have no clue about their tests cases and schedule.

Reply:
This is a UI scenario, so the only tests we have to cover this are manual tests. I'm also not sure why this didn't get caught by our manual runs.

Reply:
The Apex tests don't go through this code path?